lib/chef/provider/package/homebrew.rb

```diff
@@ -115,19 +149,23 @@ def current_installed_version
         # forward project.
         #
         # https://github.com/Homebrew/homebrew/wiki/Acceptable-Formulae#stable-versions
-        def candidate_version
-          brew_info["versions"]["stable"]
-        end
+        #
+        # @param [String] package name
+        #
+        # @returns [String] package version
+        def available_version(i)
+          p_data = package_info(i)
 
-        private
+          p_data["versions"]["stable"]
+        end
```

Comment:
this needs to return `nil` if there's no available_version (e.g. just a typo of the package name)

Reply:
Done